walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,18 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.stream.Collectors;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
     throw new UnsupportedOperationException("TODO: Implement this method.");
   }
+
+  public Boolean areAnyAttendeeBusy(Event event, MeetingRequest request) {
```

Comment:
I am a little bit confused about the name of this function. If I understand it right it is not to see "If any attendee is busy" it is more like "Is there any person in the meeting request that is already in the list of event attendees of a given event" ultimately the function that calls this function determines if attendees are busy by looking at all of the events. Please correct me if I am wrong. Thanks,

Reply:
Something that might help is changing the method signature a bit:

```java
public Boolean containsAnyAttendee(Event event, Collection<String> attendees) {
    ...
}
```

and passing in request.getAttendees().

This avoids the cognitive load required to determine what "busy" means in this context.